src/index.tsx

```diff
@@ -26,7 +39,7 @@ const store = rootStore.create(
       .then((res) => res.data as ExporterResponse)
       .catch ((error) => {
         // eslint-disable-next-line
-        logger.error(errorMsg, {response:error, userAgent, dateTime: moment(new Date()).format('DD/MM/YYYY HH:mm')});
+        logger.error(errorMsg, {response:error, ...systemInfo, dateTime: moment(new Date()).format('DD/MM/YYYY HH:mm')});
```

Comment:
can we set the data once at this location, instead of importing it every time?
https://github.com/MapColonies/image-exporter-client/blob/master/src/logger/logger.ts#L6

Reply:
Nice one, actually yes. 
But not there, will do it client-info.ts(new file)

Will re-request PR when it will be ready